types/test/filterIndexed.ts

```diff
@@ -0,0 +1,6 @@
+import * as RA from 'ramda-adjunct';
+
+const isEven = (n: number): Boolean => n % 2 === 0;
+
+RA.filterIndexed<number>((val, idx) => isEven(idx))([1, 2, 3, 4]); // $ExpectType number[]
```

Comment:
Can we have  more tests here? You've produced quite verbose type for this function, would be great if we'll have them covered with tests here.


Along with that, could we also please add some error expectations like [here](https://github.com/char0n/ramda-adjunct/blob/0b4e0f265da01aebf24b2460aeeb994d79eab5a3/types/test/allEqual.ts#L10)?

Reply:
Let me know if the update works 👍

Reply:
Looks good. thanks